plugins/eos/gs-plugin-eos.c

```diff
@@ -1730,6 +1730,10 @@ gs_plugin_refine (GsPlugin		*plugin,
 		if (gs_plugin_eos_blacklist_app_for_remote_if_needed (plugin, app))
 			continue;
 
+		/* Mark com.endlessm. apps with featured kudo. See discussion on T23152 */
+		if (g_str_has_prefix (gs_app_get_id (app), "com.endlessm."))
```

Comment:
I am not sure if we're going with this solution in the end, but if we are, perhaps verifying whether the kudo is already set makes for a good optimization (otherwise, in every operation we'd be comparing prefixes).

Reply:
That's a good idea. How often does `gs_plugin_refine` run? It seems like it's intended for batch operations that only run when refreshing.